spi/policy-monitor/policy-monitor-spi/src/main/java/org/eclipse/edc/connector/policy/monitor/spi/PolicyMonitorManager.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.policy.monitor.spi;
+
+import org.eclipse.edc.spi.entity.StateEntityManager;
+
+/**
+ * Takes care to iterate over the ongoing transfers verifying their policy, completing them whenever the policy
```

Comment:
This is a real nit, it could be shorter, "Iterates over ongoing transfers, verifying their policies and completing when the policy can no longer be evaluated"